src/main/steps/common/task-list/components/notification-banner/content.ts

```diff
@@ -326,6 +330,50 @@ const en: NotificationBannerContent = {
             },
           ],
         },
+        soaServedBannerCa: {
+          heading: 'You must give the respondent their documents',
+          sections: [
+            {
+              contents: [
+                {
+                  text: 'The court has issued your application. This means a copy of your application and other court documents are ready to give to other people in the case (the respondents)',
+                },
+                {
+                  text: 'You must give the following documents to the respondent:',
+                },
+              ],
+              links: [
+                {
+                  href: APPLICANT_STATEMENT_OF_SERVICE_RESPONDENT_DOCUMENTS,
+                  text: "View the respondent's documents",
+                },
+              ],
+            },
+            {
+              contents: [
+                {
+                  text: 'You can give the documents to the respondent or choose a person who has agreed to hand deliver them to the respondent. This can be someone you know or a professional third party(such as a process server). More information about court bailiffs can be found on <a href="https://www.gov.uk/government/publications/form-d89-request-for-personal-service-by-a-court-bailiff">GOV.UK</a>',
+                },
+                {
+                  text: '<p class="govuk-notification-banner__heading">Tell us once the respondent has been given the documents</p>',
+                },
+                {
+                  text: 'You need to submit a statement of service after the respondent has been given the documents.',
+                },
+              ],
+              links: [
+                {
+                  href: `${C9_DOWNLOAD_LINK}`,
```

Comment:
Is it an external url? If yes, does it open in new tab? tested?

Reply:
it is an external url, added the external true flag